Makefile

```diff
@@ -55,6 +56,11 @@ serve:
 devserver:
 	$(BASEDIR)/develop_server.sh restart
 
+stopserver:
+	kill -9 `cat pelican.pid`
+	kill -9 `cat srv.pid`
```

Comment:
Do we need to use ``-9``?

Reply:
Not necessary. I've revised here and in osuosl-pelican ([PR](https://github.com/osuosl/osuosl-pelican/pull/257)), too